src/OpenTelemetry.Contrib.PersistentStorage/OpenTelemetry.Contrib.PersistentStorage.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <PackageTags>$(PackageTags);Shared</PackageTags>
```

Comment:
Hmm, why do we need "Shared" nuget tag? Is this a common practice?

Reply:
We do not - will fix it.